src/Runner.Worker/action_yaml.json

```diff
@@ -120,7 +120,22 @@
                 "hashFiles(1,255)"
             ],
             "sequence": {
-                "item-type": "any"
+                "item-type": "composite-step"
```

Comment:
We need to remove `context` from `composite-steps`. Expressions aren't allowed anywhere under `composite-steps`; only certain places.

Reply:
Make sure to test and verify an expression can't be used for `id`